ads/google/imaVideo.js

```diff
@@ -1556,31 +1397,37 @@ function postMessage(data) {
  */
 export function getPropertiesForTesting() {
   return {
-    adContainerDiv,
     allAdsCompleted,
     adRequestFailed,
     adsActive,
     adsManagerWidthOnLoad,
     adsManagerHeightOnLoad,
     adsRequest,
-    bigPlayDiv,
     contentComplete,
-    controlsDiv,
     controlsVisible,
     hideControlsTimeout,
     imaLoadAllowed,
     interactEvent,
     playbackStarted,
     playerState,
     PlayerStates,
-    playPauseDiv,
-    progressLine,
-    progressMarkerDiv,
-    timeNode,
     uiTicker,
-    videoPlayer,
     hideControlsQueued,
     icons,
+    // TODO(alanorozco): Update names on test's end to pass `elements` instead.
```

Comment:
We export `elements` as their original names to illustrate how tests change. A separate PR makes this more transparent.